node/errors.js

```diff
@@ -580,3 +580,22 @@ module.exports.classify = function classify(err) {
             return null;
     }
 };
+
+// To determine whether a circuit should break for each response code.
+// TODO consider whether to keep a circuit healthy if a downstream circuit is
+// unhealthy.
+var symptoms = {
```

Comment:
This doesn't really belong in the errors module... really it should go with where we declare the codes.  Presently that's in `v2/error_response.js`.  Pulling code definition out of the v2 area into a less version-specific one has long been a discussed improvement, but I'm not sure if it's warranted.  However it would force you to take a `.../v2/...` require above, which is Working As Intended as it underscores the coupling to version-specific semantics.

Finally, where-ever it goes, it'd be great if we got static analysis test for "you covered them all" similar to our error classifer.